### ISSUE TITLE
Give full 'auto' site URL [idea]

### ISSUE THREAD

**Issue by wiz21b** [getnikola/nikola#2331]
When starting the auto mode, here's what one can see : 
```
C:\PORT-STC\PRIVATE\pilgrimp>nikola auto
Scanning posts......done!
[2016-05-03T06:17:36Z] INFO: auto: Watching files for changes...
[2016-05-03T06:17:36Z] INFO: ws4py: Using select as epoll is not available
[2016-05-03T06:17:36Z] INFO: auto: Serving HTTP on 127.0.0.1 port 8000...
```

The "problem" is that if I try to open "http://127.0.0.1:8000" in my browser, I get an error. What I need to open is actually : "http://127.0.0.1:8000/index.html".  Somehow, it took me five minutes to figure it out :(

So, I propose this in the log : 

```
C:\PORT-STC\PRIVATE\pilgrimp>nikola auto
Scanning posts......done!
[2016-05-03T06:17:36Z] INFO: auto: Watching files for changes...
[2016-05-03T06:17:36Z] INFO: ws4py: Using select as epoll is not available
[2016-05-03T06:17:36Z] INFO: auto: Serving HTTP on 127.0.0.1 port 8000...
[2016-05-03T06:17:36Z] INFO You can browse your site at http://127.0.0.1:8000/index.html
```

Best regards,

Stefan

**Comment by Kwpolska** [comment]
It looks like a Windows quirk.

**Comment by Kwpolska** [comment]
The code that did redirects was slightly broken and made really weird assumptions, fix is in #2336.

This fixes the real bug. As for the link, you may use the `-b`, `--browser` setting to open it in the browser.
